src/app/components/Figure/index.jsx

```diff
@@ -1,15 +1,28 @@
 import React from 'react';
 import { string } from 'prop-types';
+import styled from 'styled-components';
 import Caption from './Caption';
 
 const renderCaption = caption =>
   caption ? <Caption>{caption}</Caption> : null;
 
+const StyledFigure = styled.figure`
+  padding-bottom: 16px;
```

Comment:
You should use `${GEL_SPACING_DBL}` from the `src/app/lib/constants/styles.js` file, rather than use 16px directly.

Reply:
Thanks